user-bag-ui/src/modules/client/src/components/all-users/user/styles.css

```diff
@@ -0,0 +1,11 @@
+.user-view-object {
```

Comment:
I challenge you to use `sass` here instead :)

It helps with the "do not repeat yourself" principal